sentences/de-CH/_common.yaml

```diff
@@ -46,15 +46,15 @@ lists:
         out: "Fahrenheit"
 expansion_rules:
   name: "([( de | der | die | ds | s )] {name} | [( dr | ds | s | d )] {name})"
-  area: "[( idr | im | ir | ide | idä | i dä | i de )] {area}"
+  area: "[( idr | im | ir | ide | idä | i dä | i de | i der)] {area}"
```

Comment:
You should stick to rules in new hassil #830
No whitespaces areound `|` unless you expect to have a space between words. 
This will lead to duplicate spaces.